services/fhir.ts

```diff
@@ -79,33 +80,96 @@ function getInactiveSearchParam(resourceType: string) {
     return {};
 }
 
+export async function createFHIRResource(
+    resource: AidboxResource,
+    searchParams?: SearchParams
+): Promise<RemoteDataResult<any>> {
+    return service(create(resource, searchParams));
+}
+
+export function create(resource: AidboxResource, searchParams?: SearchParams): AxiosRequestConfig {
+    return {
+        method: 'POST',
+        url: `/${resource.resourceType}`,
+        data: resource,
+        ...(searchParams ? { headers: { 'If-None-Exist': searchParams } } : {}),
```

Comment:
Let's rewrite into params: searchParams instead of using If-None-Exist. Because it's aidbox-react library, and aidbox uses search params instead of If-None-Exist